src/main/java/com/hms_networks/americas/sc/extensions/util/Base64.java

```diff
@@ -0,0 +1,2597 @@
+package com.hms_networks.americas.sc.extensions.util;
+
```

Comment:
Please add a comment that this class is copyright Rob Harder and has been deemed to belong in the public domain. 
```
The source code for this class was graciously provided to the public domain by Robert Harder. He shall retain all copyrights. 
```

Reply:
Rob Harder has already added a comment pertinent to this including his website and contact information: https://github.com/hms-networks/sc-ewon-flexy-extensions-lib/blob/dev/1.7.0/src/main/java/com/hms_networks/americas/sc/extensions/util/Base64.java#L124-L130